emulator/blockchain.go

```diff
@@ -775,10 +791,12 @@ func configureTransactionValidator(conf config, blocks *blocks) *access.Transact
 }
 
 func (b *Blockchain) newFVMContextFromHeader(header *flowgo.Header) fvm.Context {
```

Comment:
Maybe rename the function to indicate that the function not only constructs a new value (`new...`), but also sets it as the active context

Reply:
Ah, that's a good point :+1: .
Maybe something like: `setFVMContextFromHeader` ?